pkg/local_workflows/report_analytics_workflow.go

```diff
@@ -224,7 +224,7 @@ func instrumentScanDoneEvent(invocationCtx workflow.InvocationContext, input wor
 	ic.SetTestSummary(toTestSummary(scanDoneEvent.Data.Attributes.UniqueIssueCount, scanDoneEvent.Data.Type))
 	ic.AddExtension("device_id", scanDoneEvent.Data.Attributes.DeviceId)
 
-	data, err := analytics.GetV2InstrumentationObject(ic)
+	data, err := analytics.GetV2InstrumentationObjectWithLogger(logger, ic)
```

Comment:
Not a super big fan of this approach myself, but open to suggestions. More thoughts [here](https://github.com/snyk/go-application-framework/pull/328/files#r2044143016).